lib/App/DuckPAN/Cmd/Test.pm

```diff
@@ -6,27 +6,44 @@ with qw( App::DuckPAN::Cmd );
 
 use MooX::Options protect_argv => 0;
 
+use Path::Tiny;
+
 option full => (
-	is          => 'ro',
-	lazy        => 1,
-	short       => 'f',
-	default     => sub { 0 },
-	doc         => 'run full test suite via dzil',
```

Comment:
I know it's totally counterintuitive, but we use tabs in the DuckPAN repo (I'm personally a fan of tabs) but we decided the IA repos would be spaces. For now let's keep this repo to Tabs. If we want, we can switch it all over in a new PR so the whole repo is consistent.

@zachthompson any thoughts/preferences here?